src/storybook/decorators/memory-stats.jsx

```diff
@@ -12,3 +12,5 @@ export function withMemoryStats(Story, options) {
   }
   return <Story />;
 }
+
+export default withMemoryStats;
```

Comment:
I am not sure what is the purpose of this component and because of it not sure if we want to export it

Reply:
This is definitely needed, as it is implementing functionality of this 
<img width="321" alt="image" src="https://user-images.githubusercontent.com/104433616/201519948-1aee2b44-792e-42d4-a57f-806462d6fd2b.png">
<img width="101" alt="image" src="https://user-images.githubusercontent.com/104433616/201519971-10c66904-492a-48d2-ae06-98cf04f3d212.png">